src/module.c

```diff
@@ -665,10 +665,11 @@ void moduleEnqueueLoadModule(sds path, sds *argv, int argc) {
     listAddNodeTail(server.loadmodule_queue, loadmod);
 }
 
-sds moduleLoadQueueEntryToLoadmoduleOptionStr(ValkeyModule *module) {
+sds moduleLoadQueueEntryToLoadmoduleOptionStr(ValkeyModule *module,
+                                              const char *config_option_str) {
     sds line;
 
-    line = sdsnew("loadmodule ");
+    line = sdsnew(config_option_str);
```

Comment:
We need to add space after "loadmodule".

```suggestion
    line = sdsnew(config_option_str);
    line = sdscatlen(line, " ", 1);
```

Reply:
wow, good catch. I hope that the tests that are currently running catch this bug.

Reply:
Maybe we don't? The tests seem to be passing.

Reply:
phew, tests caught it!